src/contracts/utils/AuthorizableUpgradeable.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
This seems like a pretty substantial change. Can we avoid needing to make this contract initializable?

Reply:
This contract is inherited into the upgradeable tokens, so nothing that is inherited can have a constructor, since any constructor will change the bytecode.